lib/internal/modules/esm/translators.js

```diff
@@ -459,6 +509,19 @@ translators.set('wasm', async function(url, source) {
   }).module;
 });
 
+// Strategy for loading a addon
+translators.set('addon', async function(url, source, isMain) {
+  emitExperimentalWarning('Importing addons');
+
+  // The addon must be loaded from file system with dlopen. Assert
+  // the source is null.
+  validateNull(source, 'source');
```

Comment:
It's not an argument, it wouldn't make sense to throw `ERR_INVALID_ARG_TYPE`. TBH I think we can simply ignore the source